apps/thehive/vol/cortex/application.conf

```diff
@@ -0,0 +1,35 @@
+play.http.secret.key="CortexTestPassword"
+play.http.session.secure=true
+
+## Necessary for local analyzer development
+docker.autoUpdate=false
+
+## ElasticSearch
+search {
+  index = cortex
+  uri = "http://elasticsearch:9200"
+}
+
+## Cache
+cache.job = 10 minutes
+
+job {
+  runner = [docker]
+}
+
+## ANALYZERS
+analyzer {
+  urls = [
+    #"https://download.thehive-project.org/analyzers.json"
+    "/opt/cortex/analyzers"
+    #"/opt/cortex/analyzers"
+  ]
+}
+
+# RESPONDERS
+responder {
+  urls = [
+    #"https://download.thehive-project.org/responders.json"
+    #"/absolute/path/of/responders"
```

Comment:
The application.conf is only provided as template. No need to change anything.

Just FYI, first link is list of docker images in dockerhub for Cortex to run (the only missing thing I was not able to implement due to Cortex related errors). Second path `/absolute/path/of/responders` is analyzer/responder path on Cortex machine if you would like to run scripts on machine rather than docker. If nothing is provided, Cortex will not see or check any analyzer/responders.

Reply:
@dominiklohmann as this is Cortex configuration file, it's not in scope for us to document further. If you see there is any value, kindly let me know and I can check/reformat documentation a bit. Otherwise can we agree on this resolved?

Reply:
If it's the default for Cortex please just add a link to upstream documentation for it at the top of the file so users can find information about it.

Reply:
Unfortunately, there is no "source-of-truth" on `application.config` files for TheHive and Cortex. Generally people go over available documentation and then going through actual Scala code to understand what key-value pairs would be useful for them in any situation.

I added reference to official sample config file.
https://github.com/tenzir/vast/blob/story/sc-36270/thehive-analyzer/apps/thehive/README.md